docker-compose.yml

```
@@ -0,0 +1,26 @@
+version: "3"
+services:
+  alfio:
+    image: alfio/alf.io:1.16
+    environment:
+      POSTGRES_PORT_5432_TCP_PORT: "5432"
+      POSTGRES_PORT_5432_TCP_ADDR: postgres
+      POSTGRES_ENV_POSTGRES_DB: alfio
+      POSTGRES_ENV_POSTGRES_USERNAME: alfio
+      POSTGRES_ENV_POSTGRES_PASSWORD: alfio
+      SPRING_PROFILES_ACTIVE: dev
+    ports:
+      - "30100:8080"
```

can we expose only the 8080 port? 

```suggestion
      - "8080:8080"
```

also, can we remove 8443? 
SSL traffic is not directly handled by the embedded jetty server, so it wouldn't work anyway...